zds/member/views.py

```diff
@@ -639,56 +639,56 @@ def login_view(request):
 
     # Redirecting user once logged in?
 
-    if "next" in request.GET:
-        next_page = request.GET["next"]
+    if 'next' in request.GET:
+        next_page = request.GET['next']
     else:
         next_page = None
-    if request.method == "POST":
+    if request.method == 'POST':
         form = LoginForm(request.POST)
-        username = request.POST["username"]
-        password = request.POST["password"]
+        username = request.POST['username']
+        password = request.POST['password']
         user = authenticate(username=username, password=password)
         if user is not None:
             profile = get_object_or_404(Profile, user=user)
             if user.is_active:
                 if profile.can_read_now():
                     login(request, user)
-                    request.session["get_token"] = generate_token()
-                    if "remember" not in request.POST:
+                    request.session['get_token'] = generate_token()
+                    if 'remember' not in request.POST:
                         request.session.set_expiry(0)
                     profile.last_ip_address = get_client_ip(request)
                     profile.save()
                     # redirect the user if needed
                     try:
                         return redirect(next_page)
                     except:
-                        return redirect(reverse("homepage"))
+                        return redirect(reverse('homepage'))
                 else:
                     messages.error(request,
-                                   _(u"Vous n'êtes pas autorisé à vous connecter "
-                                     u"sur le site, vous avez été banni par un "
-                                     u"modérateur."))
+                                   _(u'Vous n\'êtes pas autorisé à vous connecter '
+                                     u'sur le site, vous avez été banni par un '
+                                     u'modérateur.'))
             else:
                 messages.error(request,
-                               _(u"Vous n'avez pas encore activé votre compte, "
-                                 u"vous devez le faire pour pouvoir vous "
-                                 u"connecter sur le site. Regardez dans vos "
-                                 u"mails : {}.").format(user.email))
+                               _(u'Vous n\'avez pas encore activé votre compte, '
```

Comment:
En fait c'est volontaire pour garder l'homogénéité du bloc de chaine (qui sont "liées" entre elles)